lsp/client/src/extension.ts

```diff
@@ -37,10 +37,6 @@ export function activate(
         }
     };
 
-    // Get extension name
```

Comment:
have to remove the diagnostic.source which has extension title as it's value due to the popup renders it between the msg and code(link).

Reply:
This deletion makes code cleaner.  Good move!